contracts/transfer/src/lib.rs

```diff
@@ -0,0 +1,82 @@
+#![no_std]
+use dataview::Pod;
+use phoenix_abi::{Input, Note, Proof, PublicKey};
+
+type Inputs = [Input; Input::MAX];
+type Notes = [Note; Note::MAX];
+
+#[repr(C)]
+#[derive(Debug)]
+pub struct TransferArgs(Inputs, Notes, Proof);
+unsafe impl Pod for TransferArgs {}
```

Comment:
You should use the derive(Pod) in this case

Reply:
I don't think it's allow there, that's why I used the `impl`.

Reply:
We might update the macros in our fork and make it working, but I would do that in parallel.